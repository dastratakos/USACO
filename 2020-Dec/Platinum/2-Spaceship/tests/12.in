20 20 20
10111110111001011001
11001111100111011111
11000101100011100101
11101010111000001111
11101011111101111011
01111110111011011001
11111111110101100110
11011001101010111110
11101011011111111110
01010111110010011001
10001101111110011011
10001010100101101101
10011010011010111101
00110111000011111010
10111010111111000111
11111110101010111000
11101111101111111111
11001111100011111101
10001111011111111110
11110111111010111011
5 18 6 6
3 7 4 8
4 9 2 2
1 4 1 8
3 18 5 18
3 4 5 13
1 16 3 16
3 1 20 5
1 14 3 14
4 3 3 8
3 15 3 12
5 20 4 12
2 8 5 3
5 11 5 17
4 12 1 11
5 11 2 11
4 14 5 11
3 9 3 8
1 16 3 6
12 20 12 20
