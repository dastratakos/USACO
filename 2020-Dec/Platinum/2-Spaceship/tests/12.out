913483583
67417963
801798892
559377478
397608020
75224225
42959642
582938529
486918338
14604947
425114704
595276201
43286170
288543906
902090680
314812855
117311471
593413979
398075401
490408335
